.github/workflows/build_and_test.yml

```diff
@@ -2,9 +2,11 @@ name: Build and test
 
 on:
   push:
-    branches: [ python-backend ]
-  pull_request:
-    branches: [ python-backend ]
+    paths-ignore:
+      - 'python/box.tests/reports/**'
```

Comment:
At first I was afraid of an endless stream/infinite loop of automated commits, caused by e.g. changing random IDs in the git plot SVG. However, I see that this `paths-ignore` addresses it, right? The workflow will be triggered only if files **not** on this exclude list are modified?

Reply:
Yep